src/data/asean-centre-for-energy_metadata.json

```
@@ -0,0 +1,266 @@
+    "dataSource": {
+      "description": "ACE (2024). 8th ASEAN Energy Outlook (AEO8). ASEAN Centre for Energy (ACE), Jakarta.",
+      "url": "https://aseanenergy.org/",
+      "downloadAvailable": true
+    },
+    "metric": [
+      "Emissions Intensity",
+      "Capacity",
+      "Generation",
+      "Technology Mix",
+      "Absolute Emissions"
+    ],
+    "keyFeatures": {
+      "emissionsPathway": "Continued growth",
+      "energyEfficiency": "Significant improvements",
+      "energyDemand": "Significant growth",
+      "electrification": "Moderate increase",
+      "policyTypes": [
+        "Carbon price",
+        "Phaseout dates",
+        "Subsidies",
+        "Target technology shares",
+        "Other"
+      ],
+      "technologyCostTrend": "Static",
+      "technologyDeploymentTrend": "Moderate deployment of new technologies",
+      "emissionsScope": "No information",
+      "policyAmbition": "NDCs",
+      "technologyCostsDetail": "Capital costs/O&M/etc.",
+      "newTechnologiesIncluded": ["Green H2/ammonia", "SAF"],
+      "supplyChain": "No information",
+      "investmentNeeds": "Sector-level",
+      "infrastructureRequirements": "No information"
+    }
+  },
+  {
+    "id": "ACE-RAS-2024",
+    "publisher": "ASEAN Centre for Energy (ACE)",
+    "name": "Regional Aspiration Scenario (RAS)",
+    "description": "Accelerated EE, RE, and EI targets. Reaches near-term APAEC targets and conditional NDC targets.",
+    "geography": ["South East Asia"],
+    "publicationYear": 2024,
+    "pathwayType": "Normative",
```

```suggestion
    "pathwayType": "Exploratory",
```
Sorry, misclassification on my end. Will update accordingly in the excel.